mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/24 - Help Tip UI

Closes #24 

To test:
- When help tips appear (which should be in upper right of screen) clicking anywhere on the screen goes away
- Make sure that clicks pass to objects below (for example click Objectives button when tip is visible telling you to click on Objectives) so you don't have to do extra clicks to proceed
